csrankings.csv

```diff
@@ -4643,6 +4643,7 @@ Diego Pérez-Liébana,Queen Mary University of London,http://www.diego-perez.net
 Diego R. Amancio,USP-ICMC,http://icmc.usp.br/pessoas?id=10793181,WHdCNq0AAAAJ
 Diego Raphael Amancio,USP-ICMC,http://icmc.usp.br/pessoas?id=10793181,WHdCNq0AAAAJ
 Diego Vaggione,Universidad Nacional de Córdoba,http://ual.famaf.unc.edu.ar/index.php/home/diego-vaggione,NOSCHOLARPAGE
+Dieter Fox,University of Washington,https://homes.cs.washington.edu/~fox/,DqXsbPAAAAAJ

```

Comment:
He has a major leadership role at Nvidia (https://www.linkedin.com/in/dieter-fox-910aa24/). See `old/industry.csv`. He needs to have a 75% appointment at UW to be included.